python/tatr/tatrTables.py

```
@@ -0,0 +1,580 @@
+    mf = open(args.manifest, "a", newline="", encoding="utf-8")
+    mw = csv.writer(mf)
+    if new:
+        mw.writerow(["ACCESSION", "PDF", "TABLES_KEPT", "TABLES_REJECTED",
+                     "PAGES_WITH_TABLE", "MAXROWS", "MAXCOLS", "SECONDS",
+                     "STATUS"])
+
+    t_start = time.time()
+    for i, (acc, path) in enumerate(todo, 1):
+        try:
+            if not os.path.exists(path):
+                mw.writerow([acc, path, 0, 0, 0, 0, 0, 0, "missing"]); mf.flush(); continue
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)